src/test/ui/rustdoc/README.md

```diff
@@ -1,3 +1,6 @@
 This directory is for tests that have to do with rustdoc, but test the behavior
 of rustc. For example, rustc should not warn that an attribute rustdoc uses is
 unknown.
+
+All these examples are supposed to have an exact equivalent in the
```

Comment:
By "supposed to", do you mean "please do add an exact copy" or do you mean "tidy enforces that there is an exact copy"?

Reply:
"please do it", there is no enforcement. Maybe I should add one...

Reply:
Ah, in that case, could you tweak the wording so it's clear that (as of now) the situation is "please have an exact equivalent", not "tidy will enforce there is an exact equivalent"?

Reply:
Ok!